android/lib/src/androidTest/java/com/ern/api/impl/SampleActivity.java

```diff
@@ -20,8 +20,9 @@
     public boolean isBackgrounded;
     public boolean isForegrounded;
 
-    boolean didFinishFlow = false;
-    JSONObject finishFlowPayload = null;
+    public boolean didFinishFlow = false;// Indicates finishFlow() call
+    public boolean didFinish = false;// Indicates finish() call
+    public JSONObject finishFlowPayload = null;
```

Comment:
`false` and `null` are the defaults for member variables.. Since you're touching these lines here, might as well just remove the redundant values at the same time.